### ISSUE TITLE
Add tcpdump to Npcap installer

### ISSUE THREAD

**Issue by fyodor** [nmap/nmap#1341]
The main Windows tcpdump port is currently windump.exe by the WinPcap folks.  But like winpcap, it hasn't been updated in years and isn't really maintained.  It also doesn't work with our Npcap because it relies on a too-old libpcap.  But the changes to make current tcpdump work on Windows with Npcap are minor (we've done it in initial testing) and a lot of us still love tcpdump (even though it doesn't hold a candle to the amazing Wireshark feature-wise).

So the thought is that we should probably make a tcpdump.exe that uses Npcap (submitting any needed changes upstream for hopeful incorporation) and then we could include that as an optional (install by default) checkbox component.  It would be useful both to everyone who appreciates tcpdump for Windows, and also handy as a quick check that an Npcap install is working.  The tcpdump builds we have been making are less than 1mb, so fairly immaterial space wise.  And of course people can choose not to include it in the install by unchecking or using command line installer option if desired.

**Comment by guyharris** [comment]
> submitting any needed changes upstream for hopeful incorporation

Hopefully few if any will be required, given [the AppVeyor builds](https://ci.appveyor.com/project/guyharris/tcpdump), but I need to add builds with the Npcap SDK to that (we already do libpcap builds with both the WinPcap and Npcap SDKs, i.e. building against both WinPcap and Npcap packet32).

**Comment by guyharris** [comment]
> > submitting any needed changes upstream for hopeful incorporation
> 
> Hopefully few if any will be required, given [the AppVeyor builds](https://ci.appveyor.com/project/guyharris/tcpdump), but I need to add builds with the Npcap SDK to that (we already do libpcap builds with both the WinPcap and Npcap SDKs, i.e. building against both WinPcap and Npcap packet32).

OK, done in the-tcpdump-group/tcpdump@54772a25678fbe8ac836db7a02a5c98dde4e16cf; it builds OK (modulo a ton of "hey, you really might want to specify `/Qspectre` to get workarounds for Spectre inserted" warnings, which I will proceed to suppress).

**Comment by gpotter2** [comment]
Hello & sorry for bothering,
Any update on this ?

**Comment by fyodor** [comment]
No updates yet since we have been focused on dozens of other improvements (see [the Npcap changelog](https://npcap.org/changelog)), but we're definitely still planning to do this as soon as we find the time to polish up the tcpdump integration and test more and then release.

**Comment by guyharris** [comment]
> > submitting any needed changes upstream for hopeful incorporation
> 
> Hopefully few if any will be required

[Your tcpdump repository](https://github.com/nmap/tcpdump) says "This branch is 406 commits behind the-tcpdump-group:master.", without being ahead, so it doesn't look as if you have any needed changes yet.

**Comment by fyodor** [comment]
Yeah we'll integrate those change when we're ready to finish the Npcap tcpdump integration project.  We forked that branch for our proof of concept testing, but aren't really touching it until we have a few days to focus and finish that work.

**Comment by tan-wei** [comment]
If finished, it's good news for people who are familiar with `tcpdump`.
